categories/others/rules-to-better-flights.md

```diff
@@ -18,7 +18,7 @@ index:
   - do-you-know-the-best-online-accommodation-websites
   - do-you-know-how-to-get-the-most-out-of-your-credit-card
   - do-you-take-advantage-of-business-rewards-programs
-  - do-you-know-how-to-manage-qantas-bookings
+  - do-you-know-how-to-retrieve-missing-invoices-for-flights
```

Comment:
This rule url is too long, try to keep them as short as possible, ideally less than 3 words.

for this one try

missing-flight-invoices

you'll need to change the rule itself too